src/components/todoList/goalParser.test.ts

```diff
@@ -4,29 +4,38 @@ const goalObject=({
    title: 'Walk 1h daily',
     lang: 'en',
    color:  '#fff'})
+   const newData =(data)=>[...data,goalObject];
 
-const [outputObject,setOutputObject]=useState([])
+//const [outputObject,setOutputObject]=useState([])
 
-function getGoalObject(){}
+function getGoalObject(){
+  if(goalObject.title.indexOf('daily')!==-1){
+    return({
+   newData,
+   suggestion: {repetition: "daily"},
+    });}
+     else{
+      return({
+         newData
+      })
+    }
+}
 
 describe('getGoalObject function',()=>{
       it('should return repetition suggestion : daily',()=>{
-
         function fun(){
           if(goalObject.title.indexOf('daily')!==-1){
-            const newData =(data)=>[...data,goalObject]
-            setOutputObject({
+            return({
            newData,
            suggestion: {repetition: "daily"},
             });}
              else{
-              setOutputObject({
+              return({
                 newData
               })
             }
-            return outputObject;
           }
-
-        expect(getGoalObject()).toBe(fun);
+        const e=fun();
+        expect(getGoalObject()).toEqual(e);
       })
```

Comment:
It seems like `eslint` is not being used. I see a lot of code styling and convention issues